base/install_motor.sh

```diff
@@ -34,3 +36,33 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+download_github_module dls-controls pmac $PMAC_VERSION
+
+rm -f pmac/configure/RELEASE.local.linux-x86_64
+rm -f pmac/configure/RELEASE.linux-x86_64.Common
+rm -f pmac/configure/CONFIG_SITE.linux-x86_64.Common
```

Comment:
I'd prefer to not have the `-f` flag. This step will always run on a clean extraction of the module, and it would be good to get errors if any of these files are removed/renamed.